docs/manual.md

```diff
@@ -21,13 +21,43 @@ The MicroHapulator data processing workflow is broken up roughly into three phas
 2. haplotype calling
 3. analysis and interpretation
 
-In the preprocessing phase, read pairs are merged and then aligned to microhaplotype reference sequences.
+In the preprocessing phase, read pairs are merged and then aligned to MH reference sequences.
 In the haplotype calling phase, read alignments are processed one at a time to genotype and phase the SNPs of interest and to tally observed haplotypes.
 In the analysis and interpretation phase, various characteristics of the haplotype calls are examined to assess one or more forensic hypotheses.
 
-Before executing the workflow, it is important to make sure to configure MicroHapulator correctly.
+Before executing the workflow, it is important to configure MicroHapulator correctly.
 Be sure to read [the config docs](config.md) and prepare the three configuration files before beginning the workflow.
 
+Executing the MicroHapulator workflow with multiple in a stepwise fashion provides the most flexibility.
```

Comment:
with multiple samples?

Reply:
multiple command. grumble grumble.

Thanks for catching that!

Reply:
Fixed in 2b0d29e1ebb3f52fb016885858baf800b8783adc.